Lesson5/Lesson5_2/Program.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Lesson5_2
+{
+    public class Car
+    {
+        public string name;
+        private CarPart[] parts;
+
+        public Car(string name, CarPart[] parts)
+        {
+            this.name = name;
+            this.parts = parts;
+        }
+
+        public int getPartsPrice ()
```

Comment:
@MariaOsminkina Methods should start from Upper case symbol, it's not the JavaScript :)